app/test_main.py

```diff
@@ -1,3 +1,26 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAgeShouldReturnExpectedResult:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_result",
+        [
+            (-1, -1, [0, 0]),
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17]),
```

Comment:
Add test case to test against different values for `cat_age` and `dog_age` as mentioned in the checklist

Reply:
added test